apps/settings/test/unit/panels/screen_lock_passcode/screen_lock_passcode_test.js

```diff
@@ -16,18 +17,34 @@ suite('ScreenLockPasscode > ', function() {
       'shared_mocks/mock_navigator_moz_settings',
       'shared_mocks/mock_settings_listener',
       'unit/mock_settings_service',
-      'panels/screen_lock_passcode/screen_lock_passcode'
+      'panels/screen_lock_passcode/screen_lock_passcode',
+      'shared/passcode_helper'
     ];
 
     var maps = {
-      'panels/screen_lock_passcode/screen_lock_passcode': {
-        'modules/settings_service': 'unit/mock_settings_service'
+      '*': {
+        'modules/settings_service': 'unit/mock_settings_service',
+        'shared/passcode_helper': 'MockPasscodeHelper'
       }
     };
+    PasscodeHelper = {};
```

Comment:
We can setup the entire object here and make the callback of `define` return the object.